uber/models/__init__.py

```diff
@@ -652,7 +652,7 @@ def current_admin_account(self):
             return self.admin_account(cherrypy.session.get('account_id'))
 
         def admin_attendee(self):
-            if cherrypy.session.get('account_id'):
+            if getattr(cherrypy, 'session', {}).get('account_id'):
```

Comment:
When trying to run ``sep insert_admin`` we got an error about ``cherrypy.session`` not being defined.  This does the correct thing regardless of whether this property is accessed from a cherrypy request handler.